[Docs update] Update JDBC Smart Driver details (#22934)
yugabyte/yugabyte-db: docs/content/preview/integrations/flyway.md

[Flyway](https://flywaydb.org/) provides the means to manage schema changes to a YugabyteDB database, among others.

YugabyteDB has developed a plugin for better integration with Flyway. This plugin is available under the [Flyway Community DB Support](https://github.com/flyway/flyway-community-db-support) project.

## Prerequisites

To use Flyway with YugabyteDB, you need the following:

- YugabyteDB version 2.20 or later (see [Quick Start](../../quick-start/)).

- Flyway community edition version 10.12.0 or later (see [Download Flyway](https://flywaydb.org/download)).

- Flyway plugin for YugabyteDB version 10.12.0 or later (see [Maven coordinates](https://central.sonatype.com/artifact/org.flywaydb/flyway-database-yugabytedb)).

- Connection details in the Flyway configuration file `conf/flyway.conf`. Append the following at the end of the file, replacing `flyway.url` with the URL of your running YugabyteDB cluster:

  ```properties
  flyway.url=jdbc:yugabytedb://localhost:5433/yugabyte
  flyway.user=yugabyte
  flyway.password=yugabyte
  ```

>   - Alternatively, you can also use the vanilla PostgreSQL JDBC Driver instead of the YugabyteDB JDBC Smart Driver. If you use the url with prefix `jdbc:/postgresql://`, the plugin would attempt to use the corresponding driver.

Reviewer: ```suggestion
    You can also use the vanilla PostgreSQL JDBC Driver instead of the YugabyteDB JDBC Smart Driver by setting the URL prefix to `jdbc:/postgresql://`, and the plugin will attempt to use the corresponding driver.
```

Reviewer: And then reduce indent of the following code block

    ```properties
    flyway.url=jdbc:postgresql://localhost:5433/yugabyte
    flyway.user=yugabyte
    flyway.password=yugabyte
    ```

> - Add YugabyteDB JDBC Smart Driver jar in your application's classpath. The maven coordinates of the driver are given below. For its latest version, see [this mvn repository page](https://mvnrepository.com/artifact/com.yugabyte/jdbc-yugabytedb).

Reviewer: ```suggestion
- Add YugabyteDB JDBC Smart Driver jar in your application's classpath. Use the following maven coordinates; for the latest version, see the [mvn repository](https://mvnrepository.com/artifact/com.yugabyte/jdbc-yugabytedb).
```

```shell
 ./flyway migrate
```

> ## More on Flyway usage

Reviewer: ```suggestion
## Learn more
```

> Refer to [Flyway documentation](https://documentation.red-gate.com/flyway/flyway-cli-and-api/usage) for more details on using Flyway.

Reviewer: ```suggestion
Refer to the [Flyway documentation](https://documentation.red-gate.com/flyway/flyway-cli-and-api/usage) for more details on using Flyway.
```
